scripts/utils/kakarot.py

```diff
@@ -287,32 +289,63 @@ async def eth_send_transaction(
 ):
     """Execute the data at the EVM contract to on Kakarot."""
     evm_account = caller_eoa or await get_eoa()
+    nonce = await evm_account.get_nonce()
+
+    payload = {
+        "type": 0x2,
+        "chainId": KAKAROT_CHAIN_ID,
+        "nonce": nonce,
+        "gas": gas,
+        "maxPriorityFeePerGas": int(1e19),
+        "maxFeePerGas": int(1e19),
+        "to": to_checksum_address(to) if to else None,
+        "value": value,
+        "data": data,
+    }
+
+    typed_transaction = DynamicFeeTransaction.from_dict(payload)
```

Comment:
why did you change the tx type?

Reply:
Yeah mb, I didn't change the tx type, but the constructor is more specialized, we don't need this though
TypedTransaction.from_dict, with tx type == 2 will create a DynamicFeeTransaction